.travis.yml

```diff
@@ -9,14 +9,15 @@ python:
   - "2.7"
 
 install:
-  - pip install pylint tox
+  - pip install pylint tox tornado scapy
   - python setup.py install
```

Comment:
@sophron Is there a specific reason we are actually installing the package here?